spec/index.html

```diff
@@ -5688,30 +5688,45 @@ <h5>COALESCE</h5>
             <h5>NOT EXISTS and EXISTS</h5>
             <p>There is a filter operator <code>EXISTS</code> that takes a graph pattern.
               <code>EXISTS</code> returns <code>true</code>/<code>false</code> depending on whether the
-              pattern matches the dataset given the bindings in the current group graph pattern, the
-              dataset and the <a href="#defn_ActiveGraph">active graph</a> at this point in the query
-              evaluation. No additional binding of variables occurs. The <code>NOT EXISTS</code> form
-              translates into <code>fn:not(EXISTS{...})</code>.</p>
+              pattern matches the dataset given solution mapping with respect to which
```

Comment:
```suggestion
              pattern matches the dataset with respect to the solution mapping
              of the expression being <a href="#expression-evaluation">evaluated</a>.
```

Reply:
@afs I have removed that part of the sentence now.